src/lib/viewers/media/DashViewer.js

```diff
@@ -16,6 +16,9 @@ const MANIFEST = 'manifest.mpd';
 const DEFAULT_VIDEO_WIDTH_PX = 854;
 const DEFAULT_VIDEO_HEIGHT_PX = 480;
 
+const SHAKA_CODE_UNEXPECTED_NETWORK_FAILURE = 1002;
```

Comment:
You can get the error codes from shaka.util.Error.Code. So you should be able to do shaka.util.Error.Code.HTTP_ERROR instead of hard coding the number. Might be more future proof? docs at https://shaka-player-demo.appspot.com/docs/api/lib_util_error.js.html